### PR TITLE
Support spaces in MSBuild configuration

### PR DESCRIPTION
**Pull request by SharpeRAD**
Fixes an error I'm getting in a project that used to build correctly with Cake.

========================================
Build
========================================
Executing task: Build
Building Ex2007 SP3 from C:/Projects/Exchange.RemoveSection/src/RemoveSection.sln
Executing: C:/Program Files (x86)/MSBuild/14.0/Bin/amd64/MSBuild.exe /m /v:normal /p:Configuration=Ex2007 SP3 /p:Platform="Any CPU" /p:TreatWarningsAsErrors=true /target:Build "C:/Projects/Exchange.RemoveSection/src/RemoveSection.sln"
Microsoft (R) Build Engine version 14.0.23107.0
Copyright (C) Microsoft Corporation. All rights reserved.

MSBUILD : error MSB1008: Only one project can be specified.
Switch: C:/Projects/Exchange.RemoveSection/src/RemoveSection.sln

For switch syntax, type "MSBuild /help"
An error occured when executing task 'Build'.

### CODE REVIEW

File: src/Cake.Common/Tools/MSBuild/MSBuildRunner.cs [hunk 1]
```diff
@@ -61,8 +61,14 @@ private ProcessArgumentBuilder GetArguments(FilePath solution, MSBuildSettings s
             if (!string.IsNullOrWhiteSpace(settings.Configuration))
             {
                 // Add the configuration as a property.
-                var configuration = settings.Configuration;
-                builder.Append(string.Concat("/p:Configuration=", configuration));
+                if (settings.Configuration.Contains(" "))
```
Comment (patriksvensson) [inline src/Cake.Common/Tools/MSBuild/MSBuildRunner.cs]: No need for this check. Instead, always append quoted.

**Comment by patriksvensson** [comment]
@SharpeRAD Great. Left a comment for you.

**Comment by SharpeRAD** [comment]
Done :smile:

**Comment by devlead** [comment]
@SharpeRAD looks good :+1: could you rebase into 1 commit please.

**Comment by SharpeRAD** [comment]
Done :smile:

**Comment by devlead** [comment]
@SharpeRAD merged, thanks :+1:

**Pull request merged**